### ISSUE TITLE
Request: Add a "Reveal in Explorer" context menu item for Open Editors

### ISSUE THREAD

**Issue by bgashler1** [Microsoft/vscode#6687]
- VSCode Version: commit 91f14c4
- OS Version: OS X

Low priority, but it would be nice to have a "Reveal in Explorer" command within the right click menu of Open Editors.  I found myself wanting that there when I was using the latest build.  My goal was to rename a file that I had open, and I needed to see where it was in Explorer to do that.

**Comment by bpasero** [comment]
@bgashler1 there is "Show Active File in Explorer".

**Comment by bpasero** [comment]
@bgashler1 or maybe you clarify, this is a global action and you want it on an entry in the "Opened Editors" view? If you click the file it should get revealed automatically in the explorer.

**Comment by bgashler1** [comment]
@bpasero whoops, completely forgot about the auto reveal behavior.

I think why I was confused was there was a weird glitch I ran into (that I cannot reproduce now) where everything in a particular folder was an odd color.  When I'm selecting "Styles.scss" I cannot see that it's selected because it's so faint against this background.  I cannot reproduce this state after I closed VS Code and reopened (I'll open an issue if I can figure it out).

<img width="640" alt="screen shot 2016-05-23 at 11 53 12 am" src="https://cloud.githubusercontent.com/assets/11839736/15481007/4167427a-20dd-11e6-8472-3192e6f20252.png">
